e2e-tests/support/trade-finance-manager/mockTfmSsoLogin.js

```diff
@@ -0,0 +1,104 @@
+const crypto = require('crypto');
+const jws = require('jws');
+const cookieSignature = require('cookie-signature');
+const relative = require('../../tfm/cypress/e2e/relativeURL');
+
+const issueValid2faJWT = (user, sessionIdentifier) => {
+  const { _id } = user;
+  const payload = {
+    sub: _id,
+    username: user.username,
+    teams: user.teams,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    sessionIdentifier,
+  };
+  const secret = Buffer.from(Cypress.config('jwtSigningKey'), 'base64').toString('ascii');
+
+  const header = {
+    alg: 'RS256',
+    typ: 'JWT',
+  };
+  return jws.sign({
+    header,
+    payload,
+    secret,
+    encoding: 'utf8',
+  });
+};
+
```

Comment:
Could we use constants for the above values? E.g in EXIPs https://github.com/UK-Export-Finance/exip/blob/main-application-no-pdf/src/api/helpers/generate-otp/index.ts#L7

Also could or should `issueValid2faJWT` be `issueValidJWT`?